packages/runtime/test/integration/global-opcodebudget.ts

```diff
@@ -96,6 +96,10 @@ describe("TEALv6: Global Opcode Budget", function () {
 
 			const buffTx = {
 				...txnParam,
+				payFlags: {
```

Comment:
Could you explain for me about the payFlags?

Reply:
Please check this document. https://algobuilder.dev/guide/execute-transaction.html